devel/golangci-lint/Portfile

```diff
@@ -19,12 +19,12 @@ long_description    \
 categories          devel
 installs_libs       no
 license             GPL-3
-maintainers         {gmail.com:herby.gillot @herbygillot} \
+maintainers         {gmail.com:herby.gillot @herbygillot @steenzout} \
```

Comment:
If you'd like to add yourself as co-maintainer, you need to add your Github handle on its own line:

```suggestion
maintainers         {gmail.com:herby.gillot @herbygillot} \
                    {@steenzout} \
```

Ideally you could also add your obfuscated email address as well.

Reply:
@herbygillot committed suggestion. I left my email... it wouldn't be obfuscated enough. 😄

Reply:
thank you for your review! 🙏🏻